minecraft_servers/abyssal/manifest.json

```diff
@@ -15,5 +15,8 @@
     },
     "discord": {
         "server_id": "592735008427343873"
+    },
+    "brand": {
+        "primary": "#2BBFF0",
```

Comment:
Remove the extra comma to make the tests pass

Reply:
Ok I'm going to do it.